docs/resources/aws_s3_bucket.md

```diff
@@ -0,0 +1,123 @@
+---
+title: About the aws_s3_bucket Resource
+---
+
+# aws_s3_bucket
+
+Use the `aws_s3_bucket` InSpec audit resource to test properties of a single AWS bucket.
+
+To test properties of a multiple S3 buckets , use the `aws_s3_buckets` resource.
+
+<br>
+
+## Limitations
+
+S3 bucket security is a complex matter.  For details on how AWS evaluates requests for access, please see [the AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/how-s3-evaluates-access-control.html).  S3 buckets and the objects they contain support three different types of access control: bucket ACLs, bucket policies, and object ACLs.
+
+As of this writing, this resource only supports evaluating the first two, bucket ACLs and bucket policies.  Evaluating object ACLs introduces scalability concerns that are currently not well handled by the AWS API, and may be better handled via AWS mechanisms such as CloudTrail and Config.
+
+In particular, users of the `be_public` matcher should carefully examine the conditions under which the matcher will detect an insecure bucket.  See the `be_public` section under the Matchers section below.
```

Comment:
I'm not sure this is needed

Reply:
The `be_public` matcher is what we think most people will use, and assuming it does more than it actually does can make users vulnerable.